packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -312,3 +316,9 @@ export function positionTooltip(u: uPlot, bbox: DOMRect) {
 
   return { x, y };
 }
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  tooltipWrapper: css`
+    z-index: ${theme.zIndex.portal + 1} !important;
```

Comment:
Is `!important` needed here? IIFC best practices around using the `!important` declaration is that it should generally be avoided as much as possible, since it can lead to specificity and maintenance issues in our stylesheets.

Reply:
Yes, there already is a zIndex prop set on the inline style of the wrapper element itself. We need to overwrite it for tooltips since that wrapper is used across grafana and only the tooltip one needs the modification. Using `important` is generally frowned upon but I think it works for this scenario especially since tooltips will be refactors to not be global but panel based rather soon as @leeoniya mentioned earlier